examples/substrait-spark/app/build.gradle

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This project uses @Incubating APIs which are subject to change.
+ */
```

Comment:
Should we including the `examples` project in the root `settings.gradle.kts` file so that it's part of the standard build? I noticed it wasn't getting picked up by IntelliJ.

Reply:
Yes can do that; with any example, I try and make it as self-contained as possible - so it's as close to a real usage situation; and doesn't accidentally rely on being part of the main source tree.

But this shouldn't make a difference; might need to alter the CI build as well.